test/gameplay/Game.js

```diff
@@ -142,6 +142,62 @@ describe('Game', () => {
     assertGameSuccess(game.changePose(game.players[game.turn].field.monsterSlots[0].id, Card.ATTACK));
   });
 
+  it('#summon 1 tribute success', () => {
+    let card1;
+    const game = new Game([
+      {id: 'abc', name: 'F', deck: [card1 = new MonsterCard('', '', '', 5, 100, 100)]},
+    ]);
+    const handCount = game.players[game.turn].hand.length;
+    const card2 = game.players[game.turn].field.monsterSlots[0] = new MonsterCard('', '', '', 5, 100, 100);
```

Comment:
We do not limit the character count for one line since we believe that long lines are easier to read under some circumstances.

Please see our Eslint config:
https://github.com/DotLab/touhou-card-game-nodejs/blob/98569c8bc43dc3c211a42dee737da77b954faffe/.eslintrc.json#L14

The refactor did make the line shorter and easier to read nevertheless.

Comments are added.